Prohibited package name: java.time

I'm using this package in project with Scala.js and JVM cross-compilation and it crashes doobie in runtime. 

Here's exception I get:

```
java.lang.ExceptionInInitializerError
    at doobie.imports$.<init>(imports.scala:90)
    at doobie.imports$.<clinit>(imports.scala)
    at me.chuwy.doobietime.Main$.<init>(Main.scala:34)
    at me.chuwy.doobietime.Main$.<clinit>(Main.scala)
    at me.chuwy.doobietime.Main.main(Main.scala)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
    Caused by: java.lang.SecurityException: Prohibited package name: java.time
    at java.lang.ClassLoader.preDefineClass(ClassLoader.java:659)
    at java.lang.ClassLoader.defineClass(ClassLoader.java:758)
    at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
    at java.net.URLClassLoader.defineClass(URLClassLoader.java:467)
    at java.net.URLClassLoader.access$100(URLClassLoader.java:73)
    at java.net.URLClassLoader$1.run(URLClassLoader.java:368)
    at java.net.URLClassLoader$1.run(URLClassLoader.java:362)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.net.URLClassLoader.findClass(URLClassLoader.java:361)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
    at java.lang.Class.forName0(Native Method)
    at java.lang.Class.forName(Class.java:348)
    at scala.reflect.runtime.JavaMirrors$JavaMirror.javaClass(JavaMirrors.scala:555)
    at scala.reflect.runtime.JavaMirrors$JavaMirror.tryJavaClass(JavaMirrors.scala:559)
    at scala.reflect.runtime.SymbolLoaders$PackageScope$$anonfun$lookupEntry$1.apply(SymbolLoaders.scala:137)
    at scala.reflect.runtime.SymbolLoaders$PackageScope$$anonfun$lookupEntry$1.apply(SymbolLoaders.scala:126)
    at scala.reflect.runtime.Gil$class.gilSynchronized(Gil.scala:19)
    at scala.reflect.runtime.JavaUniverse.gilSynchronized(JavaUniverse.scala:16)
    at scala.reflect.runtime.SymbolLoaders$PackageScope.syncLockSynchronized(SymbolLoaders.scala:124)
    at scala.reflect.runtime.SymbolLoaders$PackageScope.lookupEntry(SymbolLoaders.scala:126)
    at scala.reflect.internal.Types$Type.findDecl(Types.scala:971)
    at scala.reflect.internal.Types$Type.decl(Types.scala:566)
    at scala.reflect.internal.SymbolTable.openPackageModule(SymbolTable.scala:335)
    at scala.reflect.runtime.SymbolLoaders$LazyPackageType$$anonfun$complete$2.apply$mcV$sp(SymbolLoaders.scala:74)
    at scala.reflect.runtime.SymbolLoaders$LazyPackageType$$anonfun$complete$2.apply(SymbolLoaders.scala:71)
    at scala.reflect.runtime.SymbolLoaders$LazyPackageType$$anonfun$complete$2.apply(SymbolLoaders.scala:71)
    at scala.reflect.internal.SymbolTable.slowButSafeEnteringPhaseNotLaterThan(SymbolTable.scala:263)
    at scala.reflect.runtime.SymbolLoaders$LazyPackageType.complete(SymbolLoaders.scala:71)
    at scala.reflect.internal.Symbols$Symbol.info(Symbols.scala:1514)
    at scala.reflect.runtime.SynchronizedSymbols$SynchronizedSymbol$$anon$1.scala$reflect$runtime$SynchronizedSymbols$SynchronizedSymbol$$super$info(SynchronizedSymbols.scala:174)
    at scala.reflect.runtime.SynchronizedSymbols$SynchronizedSymbol$$anonfun$info$1.apply(SynchronizedSymbols.scala:127)
    at scala.reflect.runtime.SynchronizedSymbols$SynchronizedSymbol$$anonfun$info$1.apply(SynchronizedSymbols.scala:127)
    at scala.reflect.runtime.Gil$class.gilSynchronized(Gil.scala:19)
    at scala.reflect.runtime.JavaUniverse.gilSynchronized(JavaUniverse.scala:16)
    at scala.reflect.runtime.SynchronizedSymbols$SynchronizedSymbol$class.gilSynchronizedIfNotThreadsafe(SynchronizedSymbols.scala:123)
    at scala.reflect.runtime.SynchronizedSymbols$SynchronizedSymbol$$anon$1.gilSynchronizedIfNotThreadsafe(SynchronizedSymbols.scala:174)
    at scala.reflect.runtime.SynchronizedSymbols$SynchronizedSymbol$class.info(SynchronizedSymbols.scala:127)
    at scala.reflect.runtime.SynchronizedSymbols$SynchronizedSymbol$$anon$1.info(SynchronizedSymbols.scala:174)
    at scala.reflect.internal.Types$TypeRef.thisInfo(Types.scala:2194)
    at scala.reflect.internal.Types$TypeRef.baseClasses(Types.scala:2199)
    at scala.reflect.internal.tpe.FindMembers$FindMemberBase.<init>(FindMembers.scala:17)
    at scala.reflect.internal.tpe.FindMembers$FindMember.<init>(FindMembers.scala:219)
    at scala.reflect.internal.Types$Type.scala$reflect$internal$Types$Type$$findMemberInternal$1(Types.scala:1014)
    at scala.reflect.internal.Types$Type.findMember(Types.scala:1016)
    at scala.reflect.internal.Types$Type.memberBasedOnName(Types.scala:631)
    at scala.reflect.internal.Types$Type.member(Types.scala:600)
    at scala.reflect.internal.Mirrors$RootsBase.getModuleOrClass(Mirrors.scala:48)
    at scala.reflect.internal.Mirrors$RootsBase.getModuleOrClass(Mirrors.scala:66)
    at scala.reflect.internal.Mirrors$RootsBase.staticModuleOrClass(Mirrors.scala:77)
    at scala.reflect.internal.Mirrors$RootsBase.staticClass(Mirrors.scala:122)
    at scala.reflect.internal.Mirrors$RootsBase.staticClass(Mirrors.scala:22)
    at doobie.util.meta$MetaInstances$$typecreator7$1.apply(meta.scala:466)
    at scala.reflect.api.TypeTags$WeakTypeTagImpl.tpe$lzycompute(TypeTags.scala:232)
    at scala.reflect.api.TypeTags$WeakTypeTagImpl.tpe(TypeTags.scala:232)
    at doobie.util.meta$Meta$$anon$1.<init>(meta.scala:209)
    at doobie.util.meta$Meta$.basic(meta.scala:208)
    at doobie.util.meta$Meta$$anon$1.xmap(meta.scala:216)
    at doobie.util.meta$MetaInstances$class.$init$(meta.scala:466)
    at doobie.util.meta$Meta$.<init>(meta.scala:171)
    at doobie.util.meta$Meta$.<clinit>(meta.scala)
    at doobie.imports$.<init>(imports.scala:90)
    at doobie.imports$.<clinit>(imports.scala)
    at me.chuwy.doobietime.Main$.<init>(Main.scala:34)
    at me.chuwy.doobietime.Main$.<clinit>(Main.scala)
    at me.chuwy.doobietime.Main.main(Main.scala)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
```

Project is pretty simple and I reproduced it without Scala.js and even without using any org.threeten classes.

```scala
import fs2._
import doobie.imports._

object Main extends App {
  val xa = DriverManagerTransactor[Task]("org.postgresql.Driver", "jdbc:postgresql:timefail", "timefail", "timefail")
  case class Item(title: String, createDate: java.sql.Timestamp)

  def insertItem(item: Item): Task[Int] = {
    val sql: Update0 = sql"INSERT INTO items (title, create_date) VALUES (${item.title}, ${item.createDate})".update
    sql.run.transact(xa)
  }

  insertItem(Item("Do somethin", new java.sql.Timestamp(144304323444L))).unsafeRun()
}
```

As soon as I add `scala-java-time` into dependencies it starts to crash.

Scala version is 2.12.1, JDK8, I tried it with scala-java-time M7 and M8.

--- a/build.sbt
+++ b/build.sbt
@@ -117,11 +117,6 @@ lazy val scalajavatime = crossProject.crossType(CrossType.Full).in(file("."))
   .jsConfigure(_.enablePlugins(TestNGScalaJSPlugin))
   .settings(commonSettings: _*)
   .jvmSettings(
-    sourceGenerators in Compile += Def.task {
-        val srcDirs = (sourceDirectories in Compile).value
-        val destinationDir = (sourceManaged in Compile).value
-        copyAndReplace(srcDirs, destinationDir)
-      }.taskValue,
     resolvers += Resolver.sbtPluginRepo("releases"),
     // Fork the JVM test to ensure that the custom flags are set
     fork in Test := true,

--- a/changes.xml
+++ b/changes.xml
@@ -9,6 +9,9 @@
   <body>
     <!-- types are add, fix, remove, update -->
     <release version="2.0.0-M9" date="2017-02-19" description="v2.0.0-M9">
+      <action dev="cquiroz" type="update" >
+        Don't include the java.time package on the jvm bundle
+      </action>
       <action dev="Daxten" type="update" >
         Attempt to guess the default timezone
       </action>

--- a/docs/src/main/tut/index.md
+++ b/docs/src/main/tut/index.md
@@ -16,7 +16,7 @@ The implementation is based on the original BSD-licensed reference implementatio
 #### Example
 
 ```tut:book
-// You can pick either java.time or org.threeten.bp package to import
+// On scala.js you can pick either java.time or org.threeten.bp package to import
 //import java.time._
 import org.threeten.bp._
 

